website/content/docs/connect/config-entries/exported-services.mdx

```diff
@@ -670,11 +723,86 @@ The following example queries the `finance` partition for the imported `payments
 ```shell-session
 $ curl 'localhost:8500/v1/health/service/payments?partition=finance'
```

Comment:
Could we account for Consul Enterprise being able to query both partitions or peers here? Could have separate `Consul Enterprise (Peers)` and `Consul  Enterprise (Partitions)` like we do above

Reply:
I like it 👍